### PR TITLE
Made a really clean way of loading images that will be reusable

### PR DESCRIPTION
**Pull request by SecondThread**
**Description**
Created a clean way of loading images in a way that is reusable that easily seperates from the UI where the images are used.

**Type of change**
Check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Fix or feature that would cause existing functionality to not work as expected
- [ ] This change requires a update in the design document

**Steps to Verify Functionality**
Passes flow and loads images like it should

**Final Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the design document (if applicable)
- [x] My changes generate no new warnings
- [x] Pulled updated master branch (if changed since branch creation) and corrected any conflicts, if any occur.

### CODE REVIEW

File: src/battlearena/BattleArena.js [hunk 1]
```diff
@@ -2,40 +2,57 @@
...
+						placeholder="Search Players" 
+						aria-label="searchPlayer" 
+						aria-describedby="basic-addon1"/>
+					<h6>Find the First Challenger Available</h6>
+					<Link to="Battleground">	
+						<button type="button" className="btn">Quickplay</button>
+					</Link>
+					<h6>Practice Against Bots</h6>
+					<Link to="TrainingArena">
+							<button type="button" className="btn">Training Arena</button>
```
Comment (blaiadam) [inline src/battlearena/BattleArena.js]: White space too long.

File: src/battlearena/BattleArena.js [hunk 2]
```diff
@@ -2,40 +2,57 @@
...
+					</Link>
+				</div>
+				<div className="column bamiddle">
+					<h2>Choose your Tank, Commander</h2>
+					<select className="dropdownMenu">
+							<option defaultValue>Select a Tank</option>
```
Comment (blaiadam) [inline src/battlearena/BattleArena.js]: Same as previous comment.

File: src/battlearena/BattleArena.js [hunk 3]
```diff
@@ -2,40 +2,57 @@
...
+							<option value="1">Child Consumer</option>
+							<option value="2">Fast Bang</option>
+							<option value="3">Biggest Gun</option>
+					</select>
+				</div>
+				<div className="column baright">
+					<h2>FORTUNA's Best and Brightest</h2>
+					<Leaderboard 
+						leaderNames={ ["We", "Need", "To", "Get", "These", "With", "An", "API", "Call", "Still"] }
```
Comment (blaiadam) [inline src/battlearena/BattleArena.js:49]: Lmao

File: src/trainingarena/TrainingArena.js [hunk 4]
```diff
@@ -26,7 +27,9 @@ class TrainingArena extends React.Component<{||}> {
                         <option value="2">Small Arena</option>
                         <option value="3">Arena I am, yes</option>
                     </select>
-                    <button type="button" className="btn">Start Battle</button>
+										<Link to="Battleground">
```
Comment (blaiadam) [inline src/trainingarena/TrainingArena.js]: White space issue.

**Review by blaiadam — CHANGES_REQUESTED** [review]
Looks great! Didn't pass flow however, I'll attach the errors.

Screenshot (29)

Screenshot (28)

**Comment by SecondThread** [comment]
Yikes, thanks for catching!

**Review by blaiadam — APPROVED** [review]
Looks good on my end.

**Pull request merged**